src/components/ConstitutionalCommitteeDetail/index.tsx

```diff
@@ -13,10 +13,10 @@ const ConstitutionalCommitteeDetail = () => {
   const { CCid } = useParams<{ CCid?: string }>();
   const { data, loading, error, statusError } = useFetch<CCDetailOVerview>(API.COMMITTEE.DETAIL_OVERVIEW(CCid || ""));
 
-  if (error && statusError !== 500) {
+  if (error && (statusError || 0) < 500) {
```

Comment:
@< 500 seems a little wide for checking if it's fine. Should it be a check for 200? Same for the lines below.

Reply:
in check error. we was check status 200( Http success). because http status is 200 then error is null